tests/test_account_page.py

```diff
@@ -22,4 +22,4 @@ def test_logout_user(web_driver: WebDriver):
 def test_shipping_address_setting(web_driver: WebDriver):
     successful_change = my_account_page.shipping_address_setting(web_driver)
     assert True == successful_change, + \
```

Comment:
To samo co wyżej czy tam niżej
```suggestion
    assert successful_change, + \
```